content/html/concepts/attributes/terms/class/class.md

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
You still need to fix the metadata. You have extra line breaks between sections and you're missing the concluding three dashes: `---`

Again, it should look like this:

```
---
Title: 'The Title'
Description: 'A brief description'
Subjects:
  - 'A subject name'
  - 'A second subject name'
  - 'An nth subject name'
Tags:
  - 'A tag'
  - 'A second tag'
  - 'An nth tag'
CatalogContent:
  - 'learn-example-course'
  - 'path/example-path'
---
```